grafana/grafana
# Traces: Span bar label

**What this PR does / why we need it**:

This PR allows you to add [additional info](https://github.com/grafana/grafana/discussions/43707) to your span bar row (Tempo/Jaeger/Zipkin). This info can be specified in the ds's settings.

The options are: None, Duration, Tag, Process.

Fixes https://github.com/grafana/grafana/issues/51124

**Special notes for your reviewer**:
To Test;
1. Go to Tempo/Jaeger/Zipkin ds settings
2. Update value for span bar label
3. Check that the label selected is reflected in the trace view

![Screenshot 2022-06-20 at 14 41 19](https://user-images.githubusercontent.com/90795735/174615309-6835a553-af1c-41c7-a8c4-8f68e48f16ec.png)
![Screenshot 2022-06-20 at 14 41 34](https://user-images.githubusercontent.com/90795735/174615312-f312fccc-ff55-42c4-81af-7d9941b998f1.png)
